analytics/integration_test/src/integration_test.cc

```diff
@@ -108,6 +108,9 @@ TEST_F(FirebaseAnalyticsTest, TestSetProperties) {
       firebase::analytics::kUserPropertySignUpMethod, "Google");
   // Set the user ID.
   firebase::analytics::SetUserId("my_integration_test_user");
+  // Initiate on-device conversion measurement.
+  firebase::analytics::InitiateOnDeviceConversionMeasurementWithEmailAddress(
+      "my_email@site.com");
```

Comment:
This isn't used. I'd recommend adding the Analytics dependency to the integration test and actually invoke the method in this test to ensure our C++ version links.

Reply:
Just to clarify, are you suggesting that I
1) add FirebaseAnalyticsOnDeviceConversion to the Podfile corresponding to the integration test, right?
2) call both the analytics.h method InitiateOnDeviceConversionMeasurementWithEmailAddress and also call the native objective C method [FIRAnalytics initiateOnDeviceConversionMeasurementWithEmailAddress:@(email_address)]

Reply:
1.  Does the integration test fail at all if you don't include the dependency in the podfile?  Can you ensure that our underlying C++ SDK is properly linked without the adding the dependency? 

2. No, I don't expect you to call the Objective-C method in the integration test.  Only the C++ Api.

The goal is just to ensure that the new method is accessible for our users.  If you call a method that isn't in the lib then the iTest will produce a link error on build, which is a test in of itself. 

I don't think there's anything that you can query from the iOS SDK to ensure that the iTest called the Objective-C version, though.